test_suite.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
```

Comment:
This won't exit with a non-zero code on error. Can you add

    set -euf -o pipefail

From https://sipb.mit.edu/doc/safe-shell/